provision/acc_provision/acc_provision.py

```diff
@@ -304,6 +308,7 @@ def config_adjust(args, config, prov_apic, no_random):
         },
         "net_config": {
             "infra_vlan": infra_vlan,
+            "gbp_pod_subnet": "%s/%s" % (cidr_split(pod_subnet)[2], cidr_split(pod_subnet)[4]),
```

Comment:
So this will always get set, right? Should be okay as long we don't have conditional overlay-only specific logic derived from it.

Reply:
This always get set, but we don't use this as a conditional for overlay solution